apps/dapp/src/components/Pages/Core/NewUI/HomeInput.tsx

```diff
@@ -55,60 +73,66 @@ export const Input = ({
   type,
   value,
   disabled,
-  suffix,
   ...props
 }: InputProps) => {
-  const renderCrypto = () => {
-    if (!crypto) {
-      return null;
-    }
+  const [isOpen, setIsOpen] = useState(false);
+  const { balance } = useWallet();
 
-    if (crypto.kind === 'value') {
-      return <Ticker>{crypto.value}</Ticker>;
-    }
+  const renderCrypto = () => {
+    if (!crypto) return null;
+    if (crypto.kind === 'value') return <Ticker>{crypto.value}</Ticker>;
     if (crypto.kind === 'select') {
-      const { cryptoOptions, defaultValue, onCryptoChange, maxSelectorItems } = crypto;
       return (
-        <InputSelect
-          options={cryptoOptions}
-          defaultValue={defaultValue}
-          onChange={onCryptoChange}
-          maxMenuItems={maxSelectorItems}
-        />
+        <>
+          <div>
+            <Ticker onClick={() => setIsOpen(true)} style={{ cursor: 'pointer' }}>
+              {crypto.selected}
```

Comment:
I'd like to add a dropdown caret SVG here but don't think we have one checked in the repo -- will add it to issues todo. 

might make sense for us to pick up an icon library at some point